infra/label_evaluator_step_functions/lambdas/evaluate_currency_labels.py

```
@@ -0,0 +1,325 @@
+                logger.info(
+                    "Loading receipt data from s3://%s/%s",
+                    batch_bucket,
+                    data_s3_key,
+                )
+                target_data = load_json_from_s3(s3, batch_bucket, data_s3_key)
+
+                image_id = target_data.get("image_id")
+                receipt_id = target_data.get("receipt_id")
+
+                if not image_id or receipt_id is None:
+                    raise ValueError("image_id and receipt_id are required in data")
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)